pytorch_toolkit/person_reidentification/config/default_config.py

```diff
@@ -30,6 +30,7 @@ def get_default_config():
     cfg.model.feature_dim = 512
     cfg.model.fpn = True
     cfg.model.fpn_dim = 256
+    cfg.model.fpn_process = 'concatenation'
```

Comment:
Currently we have three FPN-related parameters there. I suggest to join them into `cfg.model.fpn` group.

Reply:
Done.